lib/src/base-styles/_nx-form-select.scss

```diff
@@ -16,6 +17,7 @@
 }
 
 .nx-form-select {
+  @include nx-text-helpers.regular;
```

Comment:
User-agent styles specify a font for `<select>` elements which were were previously failing to override.